src/debugBridge.ts

```diff
@@ -33,6 +33,7 @@ const brokerSocket = new WebSocket(
 
 // Setup debugger socket
 debuggerSocket.on('data', (data: Buffer) => {
+    log('from function: ', data.toString('utf8').substring(0, 50));
```

Comment:
it might be better to put these log statement into if block:
```nodejs
if (LOGS_ENABLED === 'true') {
     log('from function: ', data.toString('utf8').substring(0, 50));
}
```
So there won't be extra overhead for `data.toString('utf8').substring(0, 50)` when logging is disabled